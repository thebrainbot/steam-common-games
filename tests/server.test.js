jest.unmock('../src/server');
const server = require('../src/server');

describe('Server', () => {
  test('registerPlugins registers plugins', async () => {
    await server.registerAllPlugins();
    expect(Object.keys(server.instance.registrations)).toMatchSnapshot();
  });

  test('server.instance is a Hapi server', async () => {
    expect(server.instance).toBeInstanceOf(Object);
  });
});

module.exports.instance = server;
