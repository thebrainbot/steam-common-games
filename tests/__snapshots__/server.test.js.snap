// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server registerPlugins registers plugins 1`] = `
Array [
  "graphQL",
  "graphql",
  "hapi-swagger",
]
`;
